### DISCUSSION TITLE
Welcome to Alpine Router Discussions!

### DISCUSSION THREAD

**Discussion by rehhouari** [pinecone-router/router#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by richcorbs** [comment]
Congrats on the release! I saw some of your conversations on Discord and made a mental note to follow this and see where it went. And here we are.

I've been an AlpineJS fan almost since the beginning. I love using it but only have one non-personal site that uses it.

I'm trying to wrap my head around this to see if I can use it or not. The fundamental question I'm having is why would I use a javascript router when following a link will get me to pretty much the same place almost as fast?

**Reply by rehhouari** [reply]
Thank you for your interest and feedback, I appreciate it ^^
That's a lot of experience you have, I only started learning js a few months ago and Alpine.js is the reason x)

Well for your question, it can be a matter of preference. But then again you mention the site loading fast which can be different for one user to another unless the site is just well optimized and small ^^

The usecases that comes to mind are mostly related to webapps than normal sites:
- A webapp where you want to make experience smoother, and more app-like
- Browsers' loading screen can stay empty for long if you're loading large media, which can give the user the idea that the site not responding. a solution would be to show a custom loading screen to let them wait longer
- Load just chunks of HTML instead of the entire document, and reuse them for multiple routes (views),  this can be done server side of course but in some cases you dont have a backend server or just don't want to use it

there's probably other uses but that's why I was interested in it myself. but at the end it's just a matter of preference when you're not making a webapp

**Comment by bsp982** [comment]
Very good, I develop it with someone and will use it as research. thanks
